File: iris-ld/iris-ld-proofs/src/main/java/city/smartb/iris/ldproof/crypto/rsa/RsaSignature2018LdProofSigner.java
package city.smartb.iris.ldproof.crypto.rsa;

import city.smartb.iris.crypto.rsa.signer.Signer;
import city.smartb.iris.ldproof.LdProofBuilder;
import city.smartb.iris.ldproof.signer.LdProofSigner;

public class RsaSignature2018LdProofSigner extends LdProofSigner {

    public final static String JSON_LD_RSA_SIGNATURE_2018 = "RsaSignature2018";

    public RsaSignature2018LdProofSigner(Signer signer, LdProofBuilder builder) {
        super(JSON_LD_RSA_SIGNATURE_2018, signer, builder);
    }


}

File: iris-ld/iris-ld-proofs/src/main/kotlin/city/smartb/iris/ld/ldproof/crypto/RsaSignature2018LdProofSigner.kt
package city.smartb.iris.ld.ldproof.crypto

import city.smartb.iris.crypto.dsl.signer.Signer
import city.smartb.iris.ld.ldproof.LdProofBuilder
import city.smartb.iris.ld.ldproof.signer.LdProofSigner

class RsaSignature2018LdProofSigner(signer: Signer, builder: LdProofBuilder) : LdProofSigner(
    JSON_LD_RSA_SIGNATURE_2018, signer, builder
) {
    companion object {
        const val JSON_LD_RSA_SIGNATURE_2018 = "RsaSignature2018"
    }
}
